backend/src/schema/types/type/User.gql

```diff
@@ -23,6 +23,8 @@ type User {
   #updatedAt: DateTime
   createdAt: String
   updatedAt: String
+  termsAndConditionsConfirmedAt: String
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-16T08:38:51Z" title="Friday, August 16th 2019, 10:38:51 am +02:00">Aug 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -23,6 +23,8 @@ type User {
   #updatedAt: DateTime
   createdAt: String
   updatedAt: String
+  termsAndConditionsConfirmedAt: String
```


```suggestion
```

We need `termsAndConditionsConfirmedVersion` to compare it with the current version of terms and conditions, but we don't need to expose `termsAndConditionsConfirmedAt`.